aws/cloudformation/scylla.yaml

```diff
@@ -151,6 +152,7 @@ Resources:
             - '"}]}]'
             - !Sub ', "cluster_name": "${ClusterName}", '
             - '"endpoint_snitch": "org.apache.cassandra.locator.Ec2Snitch"}, '
+            - '"start_scylla_on_first_boot": true, '
```

Comment:
how is this related to i3.metal?

Reply:
It's not
but I've change the jinja2 template, and seem like we forgot to put this change from it.

I've tested it out, and it's working as expected

Reply:
I've separated it into it's own commit

Reply:
thanks